orbit/cmd/desktop/desktop.go

```diff
@@ -477,6 +477,24 @@ func main() {
 		systray.Quit()
 	}()
 
+	// Check for the system tray icon periodically and kill the process if it's missing,
```

Comment:
Should we run this goroutine only on linux? `if runtime.GOOS == "linux" { ... }`

Reply:
We can!  My thinking was that even though this issue seems to only affect Linux, the concept of "make sure the tray icon is still there or else quit" isn't OS-specific.  But seeing as I only actually implemented a check for Linux, I'm fine not wasting the cycles on other systems.

Reply:
Done.